client/components/common/Header.js

```diff
@@ -12,6 +12,7 @@ class Header extends Component {
 
   onSignOut(){
     this.props.signOut();
+    window.location.reload();
```

Comment:
Instead of reloading the page, why don't we redirect to the homepage.

Reply:
on signout, the googleAuthInstance for google Authentication needs a browser reload otherwise it would fail when a user clicks to sign in.